crates/primitives/src/application.rs

```diff
@@ -94,10 +94,12 @@ impl fmt::Display for ApplicationSource {
 
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Application {
+    // id - Application ID created in the node for identification - see line 324 crates/context/src/lib.rs
     pub id: ApplicationId,
```

Comment:
nit~ is this comment useful? seems to me it just says `id: ApplicationId` which we already have in-code

Reply:
removed